### PR TITLE
show Q&A sessions in nav not workshops

### PR DESCRIPTION
**Pull request by tati**
As part of TFL clean up we want "Q&A sessions" in nav not "Workshops"

I also just added some comments about which view is which...let me know if you think I should remove

### CODE REVIEW

File: src/AppBar/linkSet.es6 [hunk 1]
```diff
@@ -35,12 +39,13 @@ if(! user) {
 }
 
 else {
+  // admin, mentor
```
Comment (ripleyaffect) [inline src/AppBar/linkSet.es6]: odd indention here and above

File: src/AppBar/linkSet.es6 [hunk 2]
```diff
@@ -61,9 +68,10 @@ else {
             main.push(home);
         }
         else {
+          // CP student??
```
Comment (ripleyaffect) [inline src/AppBar/linkSet.es6]: could be CP or full time CP

File: src/AppBar/linkSet.es6 [hunk 3]
```diff
@@ -11,6 +11,9 @@ if (global.__env) {
 let config = {
     workshops: {
         icon: 'users'
```
Comment (ripleyaffect) [inline src/AppBar/linkSet.es6]: can delete

File: src/AppBar/linkSet.es6 (hunk 2 above)
Comment (tati) [inline src/AppBar/linkSet.es6]: updated

File: src/AppBar/linkSet.es6 (hunk 3 above)
Comment (tati) [inline src/AppBar/linkSet.es6]: updated

File: src/AppBar/linkSet.es6 (hunk 1 above)
Comment (tati) [inline src/AppBar/linkSet.es6]: fixed

**Comment by ripleyaffect** [comment]
:+1:

**Comment by tati** [comment]
fiinalllly confirmed locally 

screen shot 2016-03-30 at 5 33 18 pm

**Pull request merged**